.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             - dependency-cache-{{ checksum "package-lock.json" }}
       - run:
           name: Install Dependencies
-          command: npm i
+          command: npm i --force #Remove force once webex version is upgraded in sdk-component-adapter 
```

Comment:
Isn't the version upgraded in sdk-component-adapter? Which version are we referring to here?

Reply:
This is Webex version mismatch which fails the npm install